modules/sfp_c99.py

```diff
@@ -0,0 +1,624 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_c99
+# Purpose:      SpiderFoot plug-in that queries c99 API
+#
+# Author:      Filip Aleksić <faleksicdev@gmail.com>
+#
+# Created:     2020-08-27
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+from sflib import SpiderFootPlugin, SpiderFootEvent
+
+
+class sfp_c99(SpiderFootPlugin):
+    """C99:Footprint,Passive,Investigate:Content Analysis:apikey:Obtain data from c99 API which offers various data (geo location, proxy detection, phone lookup, etc)."""
+
+    meta = {
+        "name": "C99",
+        "summary": "This module queries c99 API that offers various data (geo location, proxy detection, phone lookup, etc).",
+        "flags": ["apikey"],
+        "useCases": ["Footprint", "Passive", "Investigate"],
+        "categories": ["Content Analysis"],
+        "dataSource": {
+            "website": "https://api.c99.nl/",
+            "model": "COMMERCIAL_ONLY",
+            "references": ["https://api.c99.nl/api_overview", "https://api.c99.nl/faq"],
+            "apiKeyInstructions": [
+                "Visit api.c99.nl",
+                "Press shop in navigation or go to https://api.c99.nl/dashboard/shop",
+                "Press purchase key on option 'C99.NL API KEY' (you can also buy 1 year key if you want)",
+                "You should get your key on your email'",
+            ],
+            "favIcon": "https://api.c99.nl/favicon.ico",
+            "logo": "https://api.c99.nl/assets/images/logo.png",
+            "description": "C99 API service is versatile source of information. "
+            "They offer over 57 different APIs of which 10 are integrated in this module. "
+            "APIs that are integrated are subdomain finder, phone lookup, Skype resolver, "
+            "IP to Skype, firewall technology WAF detector, domain history, "
+            "IP to domains, IP geo location, proxy detector.",
+        },
+    }
+
+    opts = {"api_key": "", "response_type": "JSON", "verify": True}
```

Comment:
Any reason why we care about the response_type? Why not always have it be JSON?

Reply:
They offer two types of responses so I thought it might be good idea to give user an option to choose, but after implementing it I can see it doesn't make sense. JSON response gives so much more data and it is simpler to use